Makefile.am

```diff
@@ -48,6 +48,7 @@ OSX_QT_TRANSLATIONS = ar,bg,ca,cs,da,de,es,fa,fi,fr,gd,gl,he,hu,it,ja,ko,lt,lv,p
 
 DIST_CONTRIB = \
            $(top_srcdir)/contrib/debian/copyright \
+		   $(top_srcdir)/contrib/install_db4.sh \
```

Comment:
11702: nit: whitespaces

Reply:
Backported ShellCheck fixes in latest push!